.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Make sure these boxes are ticked before submitting a new issue:
+
+* [] I have checked that this is not a duplicate
```

Comment:
Put space between brackets.
`* [ ] I have checked that this is not a duplicate`